proto-lens-descriptors/proto-lens-descriptors.cabal

```diff
@@ -27,7 +27,7 @@ library
     , data-default-class >= 0.0 && < 0.2
     , lens-family == 1.2.*
     , lens-labels == 0.1.*
-    -- Specify an exact version of `proto-lens`, since it's tied closely
-    -- to the generated code.
-    , proto-lens == 0.2.2.0
+    -- Specify a more precise version of `proto-lens`, since it depends on
+    -- the exact definition of the Message class.
```

Comment:
It really is a pain...I'm not sure how to solve bootstrapping in that case, though.  Filed #109.